TrackingTools/KalmanUpdators/interface/Chi2MeasurementEstimatorBase.h

```diff
@@ -18,12 +18,13 @@ class Chi2MeasurementEstimatorBase : public MeasurementEstimator {
    *  The errors of the trajectory state are multiplied by nSigma 
    *  to define acceptance of Plane and maximalLocalDisplacement.
    */
-  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3.) : 
-    theMaxChi2(maxChi2), theNSigma(nSigma) {}
+  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3., float maxDisp=100.) : 
```

Comment:
was this 100 for maxdisplacement meant to be something reasonable or something "less than infinite"?
In the latter case, maybe 10000 or something would make more sense and cover all cases